pom.xml

```diff
@@ -33,6 +33,7 @@
 		<module>spring-cloud-stream-test-support</module>
 		<module>spring-cloud-stream-test-support-internal</module>
 		<module>spring-cloud-stream-configuration-metadata</module>
+		<module>spring-cloud-stream-implicit-expression-processor</module>
```

Comment:
Are you saying this should go in e.g. core? it needs to be visible at compile time (it's an annotation processor) but doesn't need to be embedded (but wouldn't hurt if it was, it's 2 classes) in the built artifact.